CHANGELOG.md

```diff
@@ -2,6 +2,34 @@
 * This file will be updated whenever a new release is put into production.
 * Any problems should be reported via the "report an issue" link in the footer of the application.
 
+## v4.2.0 - 2015-01-12
+### Important
+*This release fundamentally changes how Reservations is configured (see the ([wiki](https://github.com/YaleSTC/reservations/wiki/Configuration)) for more details). You will likely have to rework how Reservations is deployed and hosted for it to work properly when updating to this version.*
```

Comment:
I totally agree with this. I was definitely being wayyy too conservative about what makes a 'major' version, but this definitely would count. We've nominally been using SemVer for a while (I think we started this summer, technically) but haven't been rigorous about sticking to it (for example, v4.0.1 should have really been v4.1.0). In any case, I'm going to make this v5.0.0 and we'll get a polish release out during our first sprint :-).